src/seedu/addressbook/storage/StorageStub.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.storage;
```

Comment:
It’s better to add the `StorageStub` class in the `test\java\seedu\addressbook\storage` folder because it is used for testing only.